docs/wiki/EL/el-specs.md

```diff
@@ -0,0 +1,1335 @@
+
+let's start with $\xi$ :
+
+<img src="images/el-architecture/xi.png" width="800"/>
+
+This is a snapshot between blocks, like our first plot, it represents smallest slice of the potential of the  economic model additionally parameterized by $\xi$ across protocol upgrades
+
+Impact of $\xi$ on base fee:
+
+-    Inflection Point Identification: Analogous to the initial broad observation, this detailed examination reveals the nuanced shifts and inflections attributable to variations in $\xi$. The "kink" or point of inflection in the adjustment trajectory becomes particularly pronounced from this perspective.
+-    Adjustment Sensitivity: The slope of the base fee adjustment curve changes significantly beyond the inflection point. As $\xi$ values decrease, we observe a sharp increase in the rate of fee adjustments, indicating heightened sensitivity.
+-    Step Width Variability: Increasing $\xi$ results in broader step widths, indicating more gradual fee adjustments. Conversely, decreasing $\xi$ leads to narrower steps and more volatile fee changes.
+-    Linear Trend within Target Range: The central portion of the curve, particularly highlighted by the light green line for the current $\xi$ value, showcases a mostly linear trend in fee adjustments as transactions approach or slightly exceed the gas target.
+
+Next, we turn our attention to the elasticity multiplier ($\rho$), another pivotal constant in Ethereum's economic model that directly influences the flexibility and responsiveness of gas limit adjustments. To comprehensively understand its impact, we explore a range of values for $\rho$ from 1 to 6 in conjunction with variations in the base fee max change denominator ($\xi$).
+
+<img src="images/el-architecture/rho-xi.png" width="800"/>
+
+Impact of $\rho$ and $\xi$ on Base Fee :
+
+-    Moment-to-Moment Analysis: Similar to our initial observations, this plot offers a granular view into how adjustments in $\rho$ and $\xi$ shape the economic model's behavior on a per-block basis, especially in the context of protocol upgrades.
+-    Distinct Influence of $\rho$: Each subplot represents the nuanced effects of varying $\rho$ values. As the elasticity multiplier, $\rho$ notably shifts the inflection point in the base fee adjustment curve, highlighting its critical role in tuning the network's responsiveness to transaction volume fluctuations.
+-    Interplay Between $\rho$ and $\xi$: The elasticity multiplier ($\rho$) not only moves the inflection point but also modulates the sensitivity of adjustments attributable to changes in the base fee max change denominator ($\xi$). This interaction underscores the delicate balance Ethereum maintains to ensure network efficiency and stability amidst varying demands.
+
+<img src="images/el-architecture/gas-header.png" width="800"/>
+
+TODO Blob fee charts
+
+Client Code :
```

Comment:
again, I would only focus on cancun and move the client codebase references to el-client-architecture doc